tools/sdk/lib/compare/sdk-compare-libs

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Some comments explaining this script would help.

Reply:
moved to #4855 (improved and documented)